md/languages.md

```diff
@@ -25,23 +27,20 @@ For the web mobile version:
 
 ## Add a language to Bonita BPM Portal
 
-::: warning
-We are currently updating the translation process. Among other things this update impacts the Crowdin project structure. Instruction below might be temporarily inaccurate.
-:::
-
 ::: info
 Instructions below assume that the language you want to add is already available in the community translation project. If translation is not available see [Translate Bonita BPM Portal](#Translate-Bonita-BPM-Portal) for instruction about how to collaborate to a new or ongoing translation.
 :::
 
-Instructions below explain how to add support for a language to Bonita BPM Portal. Steps below involve using content created by Bonitasoft community and therefore not officially supported.
+Instructions below explain how to add a language to Bonita BPM Portal. Steps below involve using content created by Bonitasoft community and therefore not officially supported.
 
-1. Go to [Bonita BPM translation project](http://translate.bonitasoft.org/)
-1. Click the **_Download_** icon to download the zip archive file containing the translations into all the available languages. The archive is called `bonita-bpm-60.zip` even for Bonita BPM 7 versions.
-1. Go to the `master/bonita-web/portal` folder.
-1. Find and extract the `.po` files for the language you want to add (several files for a single language). Each `.po` file has a language indicator and a locale indicator. For example, the files for the Brazilian Portuguese language end with `pt_BR.po`.
-1. For Subscription editions, you also need to get the files from `master/bonita-web-sp/portal` folder.
-1. Copy the `.po` files to a new folder of your choice, preferably in a sub-folder inside your bundle. E.g. `[TOMCAT_INSTALL_FOLDER]/i18n/`. For Bonita BPM Studio, Tomcat installation folder is located in `/workspace/tomcat`.
+1. Go to [Bonita BPM translation project](http://translate.bonitasoft.org/).
+1. Select the language you are interested in.
+1. Browse the file tree to `7.3.x/bonita-web/portal` folder.
```

Comment:
"7.3.x" may be replaced by ${varVersion} as explained here: https://github.com/bonitasoft/bonita-doc/blob/7.3/CONTRIBUTING.md since it applies to 7.3+ versions